scripts/build/build-utils.ps1

```diff
@@ -86,18 +86,29 @@ function Invoke-MSBuild (
 }
 
 # Tests
-function Invoke-UnitTests([string]$binPath) {
+function Invoke-UnitTests([string]$binPath, [bool]$failsOnNotTest) {
     Write-Header "Running unit tests"
 
     $escapedPath = $binPath -Replace '\\', '\\'
 
     $testFiles = @()
+    $testDirs = @()
     Get-ChildItem ".\src" -Recurse -Include "*.UnitTest.dll" `
         | Where-Object { $_.DirectoryName -Match $escapedPath } `
-        | ForEach-Object { $testFiles += $_ }
+        | ForEach-Object {
+            $testFiles += $_
+            $testDirs += $_.Directory
```

Comment:
Are the test adapters now automatically copied into the test directory?

Reply:
Don't know for all of them but MsTest 2 - which we use for some time now - does.